sources/packages/web/src/views/institution/locations/active-applications/LocationActiveApplicationSummary.vue

```diff
@@ -96,16 +96,28 @@ export default {
         );
     };
 
+    const loadLocationDetails = async () => {
+      locationDetails.value =
```

Comment:
I believe that this value is retrieved only to populate the header. It seems something to be retrieved from the state. Thoughts?

Reply:
agree.. maybe I will create a technical debt for it, because it need to be updated in all these pages 
![image](https://user-images.githubusercontent.com/77353155/163873200-97690f03-61b9-4cbb-833b-99b8cf297d00.png)